proto/source.proto

```diff
@@ -20,8 +20,11 @@ message StreamRequest {
 
 message StreamContext {
 	bool isInteractivitySupported = 1;
+	// kubeConfig is path to kubeConfig on local FS
 	string kubeConfig = 2;
```

Comment:
This isn't used anywhere.

Reply:
What do you mean by this comment? Shouldn't it be used? We need to pass kubeconfig to source plugin somehow, right? it would be good to pass it explicitly in grpc api same as for executors.

Reply:
I added a new field (`[]byte`) in the spirit of not breaking existing API.

Since we don't have any 3rd party plugins depending on existing `kubeconfig` field, we can think about making breaking change.

https://github.com/kubeshop/botkube/blob/069937a024add2f4a35a9c5455360702a7715d1b/proto/source.proto#L21-L28

Reply:
> I added a new field ([]byte) in the spirit of not breaking existing API.

Let's break it, we already had multiple breaking changes 👍